schemas/ros1/TrianglesMarker.msg

```diff
@@ -0,0 +1,49 @@
+# Generated from TrianglesMarker by @foxglove/schemas
+
+# Timestamp of the marker
+time timestamp
+
+# Frame of reference
+string frame_id
+
+# Namespace into which the marker should be grouped. A marker will replace any prior marker on the same topic with the same `namespace` and `id`.
+string namespace
+
+# Identifier for the marker. A marker will replace any prior marker on the same topic with the same `namespace` and `id`.
+string id
+
+# Length of time (relative to `timestamp`) after which the marker should be automatically removed. Zero value indicates the marker should remain visible until it is replaced or deleted.
+duration lifetime
+
+# Whether the marker should keep its location in the fixed frame (false) or follow the frame specified in `frame_id` as it moves relative to the fixed frame (true)
+bool frame_locked
+
+# Additional user-provided metadata associated with the marker. Keys must be unique.
+foxglove_msgs/KeyValuePair[] metadata
+
+# 0-1-2, 3-4-5, ...
+uint8 TRIANGLE_LIST=0
+
+# 0-1-2, 1-2-3, 2-3-4, ...
+uint8 TRIANGLE_STRIP=1
```

Comment:
-1

Reply:
This comment is not helpful. Can you please provide some explanation or reasoning so that an intelligent response can be made? I think we previously discussed this briefly but I don't remember the reasoning for removing this.

Reply:
https://github.com/foxglove/schemas/pull/17#issuecomment-1132255935